crates/aptos/src/op/key.rs

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    common::{
+        types::{CliCommand, EncodingType, Error, KeyType},
+        utils::{prompt_yes, to_common_success_result},
+    },
+    CliResult,
+};
+use aptos_crypto::{ed25519::Ed25519PrivateKey, x25519, Uniform, ValidCryptoMaterial};
+use rand::SeedableRng;
+use std::{fs::File, io::Write, path::PathBuf, process::exit};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+pub enum KeyTool {
+    Generate(GenerateKey),
+}
+
+impl KeyTool {
+    pub async fn execute(self) -> CliResult {
+        match self {
+            KeyTool::Generate(generate) => generate.execute(),
```

Comment:
feels like generate should make key and key.pub not just key?

Reply:
can we also delete stuff along the way, like ... generate key in config? :P
also another good one to kill is the seed peers config file generator

Reply:
I don't want to delete them until afterwards, since they are used in tutorials.

If we change the tutorials and guides, we can delete the tools.

Reply:
Now generates both.